### PR TITLE
Add js-beautify checks and convenience make target

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/69082

### CODE REVIEW

**Comment by okurz** [comment]
@kalikiana SCNR ;)

I hope you forgive me :)

**Comment by okurz** [comment]
anyone feel free to suggest rules from https://github.com/beautify-web/js-beautify#options that we should apply instead of the default

**Comment by kalikiana** [comment]
> @kalikiana SCNR ;)
> 
> I hope you forgive me :)

Only if you put the ticket in the description ;-)

My only slight concern would be availability of editor integration (eslint integrates with various tools, JS Beautifier doesn't even mention it anywhere obvious as far as I can see). The availability of a Python API is awesome, though!

File: assets/javascripts/admin_groups.js [hunk 1]
```diff
@@ -273,35 +272,37 @@ function saveReorganizedGroups() {
         $('#reorganize_groups_error').show();
         $('#reorganize_groups_progress').hide();
         $('#reorganize_groups_error_message').text(thrownError ? thrownError : 'something went wrong');
-        $('html, body').animate({scrollTop: 0}, 1000);
+        $('html, body').animate({
+            scrollTop: 0
+        }, 1000);
```
Comment (Martchus) [inline assets/javascripts/admin_groups.js:277]: The previous  style was better.

File: assets/javascripts/admintable.js [hunk 2]
```diff
@@ -31,7 +35,9 @@ function showAdminTableRow(row) {
     }
 
     // scroll to the new row
-    $('html').animate({scrollTop: $(row.node()).offset().top}, 250);
+    $('html').animate({
+        scrollTop: $(row.node()).offset().top
+    }, 250);
```
Comment (Martchus) [inline assets/javascripts/admintable.js:40]: The previous style looked better.

File: assets/javascripts/admintable.js [hunk 3]
```diff
@@ -40,7 +46,9 @@ function addAdminTableRow() {
     // add new row
     var newRow = adminTable.row.add(adminTable.emptyRow);
     var newRowIndex = newRow.index();
-    adminTable.rowData[newRowIndex] = jQuery.extend({isEditing: true}, adminTable.emptyRow);
+    adminTable.rowData[newRowIndex] = jQuery.extend({
+        isEditing: true
+    }, adminTable.emptyRow);
```
Comment (Martchus) [inline assets/javascripts/admintable.js:51]: The previous style looked better.

File: assets/javascripts/admintable.js [hunk 4]
```diff
@@ -166,7 +174,10 @@ function getAdminTableRowData(trElement, dataToSubmit, internalRowData) {
                     var key = this.substr(0, equationSignIndex);
                     var val = this.substr(equationSignIndex + 1);
                     settingsToSubmit[key] = val;
-                    internalRowSettings.push({key: key, value: val});
```
Comment (Martchus) [inline assets/javascripts/admintable.js:169]: Seems like it really doesn't like writing objects in a compact way. Maybe there's a rule/threshold to change that?

File: assets/javascripts/admin_groups.js (hunk 1 above)
Comment (okurz) [reply assets/javascripts/admin_groups.js:277]: Added the CLI option `--brace-style=collapse,preserve-inline` to preserve the style without line breaks. It addes a bit of spaces though, I hope that's ok:

```
-        $('html, body').animate({scrollTop: 0}, 1000);
+        $('html, body').animate({ scrollTop: 0 }, 1000);
```

File: assets/javascripts/admintable.js (hunk 2 above)
Comment (okurz) [reply assets/javascripts/admintable.js]: see https://github.com/os-autoinst/openQA/pull/3268#discussion_r456940506

File: assets/javascripts/admintable.js (hunk 3 above)
Comment (okurz) [reply assets/javascripts/admintable.js]: see https://github.com/os-autoinst/openQA/pull/3268#discussion_r456940506

File: assets/javascripts/admintable.js (hunk 4 above)
Comment (okurz) [reply assets/javascripts/admintable.js:169]: that would be https://github.com/os-autoinst/openQA/pull/3268#discussion_r456940506 with the only additional of additional spaces if that's ok

File: assets/javascripts/admin_user.js [hunk 5]
```diff
@@ -1,5 +1,9 @@
 function setup_admin_user() {
-    window.admin_user_table = $('#users').DataTable({order: [[0, 'asc']]});
+    window.admin_user_table = $('#users').DataTable({
+        order: [
+            [0, 'asc']
+        ]
+    });
```
Comment (Amrysliu) [inline assets/javascripts/admin_user.js:6]: This style is ok for me. But I prefer the previous one.

**Pull request merged**